toolkit/tools/internal/file/file.go

```diff
@@ -101,6 +103,9 @@ func CopyDir(src, dst string, newDirPermissions, mergedDirPermissions, childFile
 		if err != nil {
 			return err
 		}
+		if !isDstDir {
+			return fmt.Errorf("destination exists but is not a directory (%s)", dst)
+		}
 		if isDstDir {
```

Comment:
nit: Remove the `if isDstDir {` check. It is redundant.

Reply:
removed.